api/python/cellxgene_census/tests/experimental/pp/test_hvg.py

```diff
@@ -83,7 +108,11 @@ def test_hvg_vs_scanpy(
         )
         kwargs["batch_key"] = "the_batch_key"
 
-    scanpy_hvg = sc.pp.highly_variable_genes(adata, inplace=False, **kwargs)
+    try:
+        scanpy_hvg = sc.pp.highly_variable_genes(adata, inplace=False, **kwargs)
+    except ZeroDivisionError:
+        pytest.skip("ScanPy generated an error, likely due to batches with 1 sample")
```

Comment:
Maybe add a comment that explains a bit better the issue here? We could even link the issue you created for this ticket, since that seems clear enough.

Reply:
the ticket / issue has nothing to do with the above issue (the ticket is a crash in our code, the above fix is to accommodate the failure of ScanPy on over-batched test data).

Reply:
and I will add an additional comment